addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxDiscoveryService.java

```diff
@@ -142,6 +146,27 @@ public void onDataFetched(ThingUID bridge, LanHostsConfig hostsConfig) {
                 }
             }
         }
+
+        if (airPlayDevices != null) {
+            // AirPlay devices
+            for (AirMediaReceiver device : airPlayDevices) {
+                String name = device.getName();
+                Boolean videoCapable = device.isVideoCapable();
+                logger.debug("AirPlay Device name {} video capable {}", name, videoCapable);
+                if ((name != null) && !name.isEmpty() && Boolean.TRUE.equals(videoCapable)) {
+                    String uid = name.replaceAll("[^A-Za-z0-9_]", "_");
+                    thingUID = new ThingUID(FreeboxBindingConstants.FREEBOX_THING_TYPE_AIRPLAY, bridge, uid);
+                    if (thingUID != null) {
```

Comment:
@lolodomo I should have just fixed in all those other bindings, see https://github.com/openhab/openhab2-addons/pull/2152 :-)